whois-query/src/test/java/net/ripe/db/whois/query/planner/AbuseCFinderTest.java

```diff
@@ -52,6 +55,9 @@ public void getAbuseContacts_oneOrgReferenceNoAbuseC() {
         final String abuseContact = subject.getAbuseContact(inetnum);
 
         assertThat(abuseContact, is(nullValue()));
+
+        verify(maintainers).isRsMaintainer(ciSet());
```

Comment:
Should we test maintainers in this class? Can we put extra checks into MaintainerTest instead?